components/card/overview.md

```diff
@@ -62,11 +62,12 @@ The Card provides various parameters that allow you to configure the component:
 
 | Parameter   | Type | Description |
 | ----------- | ----------- | -------|
-| `Width` | `string` | defines width of the component.
 | `Class` | `string` | the CSS class that will be rendered on the main wrapping element of the Card.
```

Comment:
The content of the `Description` column requires a minor update. The first letter must be capitalized on all rows. See the `components/carousel/overview.md` file in this PR, for example.